docs/questions.rst

```diff
@@ -419,3 +419,44 @@ tmt example plan (L2 metadata):
 
 See the :ref:`/spec/plans/discover/fmf` plugin documentation for
 more details.
+
+
+Migrating provision.fmf
+------------------------------------------------------------------
+
+The ``provision.fmf`` file is used to specify storage and network
+devices. In this migration, the contents of the ``provision.fmf``
+file is moved to the ``provision`` step under ``hardware``
+specification.
+
+``provision.fmf`` example:
+
+.. code-block:: yaml
+
+    standard-inventory-qcow2:
+        qemu:
+            drive:
+                - size: 10737418240
+                - size: 10737418240
+                - size: 10737418240
+
+tmt example plan (L2 metadata):
+
+.. code-block:: yaml
+
+    provision:
+        how: virtual
+        hardware:
+            disk:
+                - size: ">10GiB"
+                - size: ">10GiB"
+                - size: ">10GiB"
+
+See the :ref:`/spec/hardware/disk` and :ref:`/spec/hardware/network`
+documentation for more details about these hardware specifications
+in tmt plans.
+
+If you were using ``provision.fmf`` with Testing Farm, check out
+the Testing Farm docs on this HW requirement,
+https://docs.testing-farm.io/general/0.1/test-request.html,
```

Comment:
@janhavlin please see how tmt developers write links to other pages, you can find example in this file

Reply:
You forgot to link it. :)

But the way I did it (just paste the link without any rst formatting) seems to be common, [example 1](https://github.com/teemtee/tmt/blob/main/docs/contribute.rst?plain=1#L254), [example 2](https://github.com/teemtee/tmt/blob/main/docs/examples.rst?plain=1#L414), [example 3](https://github.com/teemtee/tmt/blob/main/README.rst?plain=1#L540).

But yeah, the most common way is to use:
```
See `TF docs`__.

__ link
```

Reply:
Fixed, yeah, looks way better.